lib/model/SendImage.dart

```diff
@@ -4,12 +4,12 @@ import '../interface/JsonObject.dart';
 import '../model/ProductImage.dart';
 
 class SendImage extends JsonObject {
-  OpenFoodFactsLanguage lang;
+  OpenFoodFactsLanguage? lang;
 
   // ignored for json
-  Uri imageUrl;
+  Uri? imageUrl;
```

Comment:
at some point (not necessarily now), we probably should see which of those fields should be non-nullable. e.g. we should not allow SendImage to be called with a null imageUrl or a null barcode.

Reply:
I agree. The work here has room for improvements but tweaking the code base is outside the scope of the PR. It would be too much work. Let's improve it as we go on.